modules/installation-guide/partials/proc_using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create.adoc

```diff
@@ -0,0 +1,33 @@
+[id="using-the-operator-to-configure-the-number-of-workspaces-that-a-user-can-create_{context}"]
+= Using the Operator to configure the number of workspaces that a user can create 
+
+This procedure describes how to use the Operator to configure the number of workspaces that a user can create. 
```

Comment:
Introductory sentence is just repeating the headline.

Reply:
@tstastna Hello Tereza. Try to ask the dev you work on this with about how this setup could enhance user experience and add it there :)

Reply:
"This procedure describes how to ..." might be classified as self-referential text according to ibm_style.
I haven't read anything outside of the diff view, but this sentence could also be reworded into a general statement of fact like so:
"ConfigMap enables you/admin/users to configure the number of workspaces that a user (any user? you/reader?) can create."